modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/admin/throttlingpolicy/GetThrottlingPoliciesTestCase.java

```diff
@@ -0,0 +1,154 @@
+/*
+ *
+ *   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an
+ *   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *   KIND, either express or implied.  See the License for the
+ *   specific language governing permissions and limitations
+ *   under the License.
+ *
+ */
+
+package org.wso2.am.integration.tests.restapi.admin.throttlingpolicy;
+
+import com.google.common.io.Files;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.am.integration.clients.admin.api.dto.ThrottlePolicyDetailsDTO;
+import org.wso2.am.integration.clients.admin.api.dto.ThrottlePolicyDetailsListDTO;
+import org.wso2.am.integration.test.helpers.AdminApiTestHelper;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationBaseTest;
+import org.wso2.am.integration.test.utils.base.APIMIntegrationConstants;
+import org.wso2.am.integration.test.utils.http.HTTPSClientUtils;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.integration.common.admin.client.UserManagementClient;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.wso2.am.integration.test.utils.generic.APIMTestCaseUtils.encodeCredentials;
+
+public class GetThrottlingPoliciesTestCase extends APIMIntegrationBaseTest {
+    private final String getThrottlePoliciesResource = "/throttling/policies/search";
+    private final  String appPolicyName="50PerMin";
+    private final String subPolicyName ="Gold";
```

Comment:
Constants should be in snake case

Reply:
Fixed with 1531fb99b70736bf5c89cd0a0b5c3ae270915472